cmd/beekeeper/cmd/check_gc.go

```diff
@@ -52,11 +58,23 @@ func (c *command) initCheckGc() *cobra.Command {
 				seed = random.Int64()
 			}
 
+			if runReserve {
+				return gc.CheckReserve(cluster, gc.Options{
+					NodeGroup:        "nodes",
+					StoreSize:        c.config.GetInt(optionNameDbCapacity),
+					StoreSizeDivisor: c.config.GetInt(optionNameDivisor),
```

Comment:
I would add a `ReserveSize` flag here

Reply:
I think adding this is still necessary, since we might need to fine tune the reserve size in `bee` integration testing and having it is a flag here would make our lives easier to tweak this